host-configs/CVX/cvx-wsl.cmake

```diff
@@ -0,0 +1,17 @@
+site_name(HOST_NAME)
+set(CONFIG_NAME "cvx-wsl" CACHE PATH "" FORCE)
+message( "CONFIG_NAME=${CONFIG_NAME}" )
+
+set(CMAKE_C_COMPILER "/usr/bin/gcc" CACHE PATH "" FORCE)
+set(CMAKE_CXX_COMPILER "/usr/bin/g++" CACHE PATH "" FORCE)
+set(CMAKE_Fortran_COMPILER "/usr/bin/gfortran" CACHE PATH "" FORCE)
+set(ENABLE_FORTRAN OFF CACHE BOOL "" FORCE)
+
+set(ENABLE_MPI ON CACHE PATH "" FORCE)
+set(MPI_C_COMPILER "/usr/bin/mpicc" CACHE PATH "" FORCE)
+set(MPI_CXX_COMPILER "/usr/bin/mpicxx" CACHE PATH "" FORCE)
+set(MPI_Fortran_COMPILER "/usr/bin/mpifort" CACHE PATH "" FORCE)

```

Comment:
Same, define `GCC_ROOT` and `MPI_ROOT` and rely on a common file to define the compilers for you.

Reply:
oh ok, missed that file...

Reply:
ok, should be good now